File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/04.For-Loop/03-additional/06.bills.js
function bills(input) {

    const billForWater = 20;
    const billForNet = 15;

    const period = Number(input[0]);
    let index = 1;

    const totalBillForWater = billForWater * period;
    const totalBillForNet = billForNet * period;
    let totalBillForElecrticity = 0;
    let totalAnotherBills = 0;

    for (let i = 1; i <= period; i++) {
        let currentBillForElectricity = Number(input[index++]);
        totalBillForElecrticity += currentBillForElectricity;
        // • за други – събират се тока, водата и интернета за месеца и към сумата се прибавят 20%.
        totalAnotherBills = (totalBillForElecrticity + totalBillForWater + totalBillForNet) * 1.2;
    }

    const averageBillsPerMonth = (totalBillForElecrticity + totalBillForWater + totalBillForNet + totalAnotherBills) / period;
    console.log(`Electricity: ${totalBillForElecrticity.toFixed(2)} lv`);
    console.log(`Water: ${totalBillForWater.toFixed(2)} lv`);
    console.log(`Internet: ${totalBillForNet.toFixed(2)} lv`);
    console.log(`Other: ${totalAnotherBills.toFixed(2)} lv`);
    console.log(`Average: ${averageBillsPerMonth.toFixed(2)} lv`);
}
bills(["5"
    , "68.63"
    , "89.25"
    , "132.53"
    , "93.53"
    , "63.22"]);

File: Python_Basics/Homeworks-And-Labs/04.for_loop/03.additional/06.bills.py
bill_for_water = 20
bill_For_net = 15

period = int(input())

total_bill_for_water = bill_for_water * period
total_bill_For_net = bill_For_net * period
total_bill_for_electricity = 0
total_another_bills = 0

for i in range(period):
    current_bill_for_electricity = float(input())
    total_bill_for_electricity += current_bill_for_electricity
    # • за други – събират се тока, водата и интернета за месеца и към сумата се прибавят 20%.
    total_another_bills = (total_bill_for_electricity + total_bill_for_water + total_bill_For_net) * 1.2

average_bills_per_month = (total_bill_for_electricity + total_bill_for_water + total_bill_For_net + total_another_bills) / period
print(f'Electricity: {total_bill_for_electricity:.2f} lv')
print(f'Water: {total_bill_for_water:.2f} lv')
print(f'Internet: {total_bill_For_net:.2f} lv')
print(f'Other: {total_another_bills:.2f} lv')
print(f'Average: {average_bills_per_month:.2f} lv')
